driver/driver.go

```diff
@@ -27,6 +28,11 @@ var (
 	ErrModeNotSupported = errors.NewKind("transform mode not supported")
 )
 
+const (
+	// ManifestLocation location of the manifest file in the driver image.
```

Comment:
Since SDK controls the build and the `Dockerfile`, there is really no way to change this.

But from SDK perspective, this is a default value that may be overridden by tests.

So probably it makes sense to leave it as a `ManifestLocation` so users will have no doubt where the manifest is.

Reply:
Hmm, now I think it will be better to change the `server.ManifestLocation`  from `var` to `const` (as a separate PR). The tests can use a different mechanism for this.